src/shared/components/ui/Button/Button.stories.tsx

```diff
@@ -1,52 +1,65 @@
 import type { Meta, StoryObj } from "@storybook/react";
 
-import { Button } from "./Button";
+import { Button } from ".";
 
 const meta: Meta<typeof Button> = {
   component: Button,
-  parameters: {
-    componentSubtitle: "A simple button component",
+  parameters: { componentSubtitle: "A simple button component" },
+  args: {
+    variant: "default",
+    size: "default",
+    children: "Click Me",
+    disabled: false,
+  },
+  argTypes: {
+    variant: {
+      control: { type: "select" },
+      options: ["default", "outline", "info", "success", "error"],
+    },
+    size: {
+      control: { type: "radio" },
+      options: ["sm", "default", "lg"],
+    },
+    asChild: { table: { disable: true } },
```

Comment:
If you remove this argTypes, doesn't TypeScript already infer the options you can choose in the story?

Reply:
No, by using tailwind variants it does not seem to auto-recognise the options types (select or radio) and instead presents a free text field